### DISCUSSION TITLE
Did this project survive the war?

### DISCUSSION THREAD

**Discussion by lordpoee** [tenacityteam/tenacity-legacy#134] [Q&A] [answered]
I read this article,
https://www.theregister.com/2021/07/07/tenacity_maintainer_quits_4chan_harassment/

Should I bother cloning?

**Comment by fossdd** [comment]
You can clone tenacity: it doesn't need much, just some space (~200 MB) and some minutes.

But I guess there is not the idea to end that project and remove the whole source code or something like that.

**Comment by Nico-Yellow** [comment, marked as answer]
Pretty sure the project will survive, other people are onboard and motivated.

As of now, you should not have any problems, this is more or less Audacity but without their recent telemetry update.

**Comment by MrGucciL** [comment]
No this fork is dead. Sneedacity is the one moving forward.

**Reply by drankinatty** [reply]
Appears to be a troll. I'll stick with this project -- thank you. Who needs a sneed?
